src/trackCopiedText.js

```diff
@@ -1,18 +1,57 @@
-/**
- * Tracking text copied by the user.
- * 
- * Adds event listener that monitors all copy events and sends text copied at the time as custom event.
- */
-
- export default (subscribe) => {
-    
-    const processCopyEvent = () => {
-        let copiedItemText = getSelectedText();
-        var eventData = ["trackEvent","User interaction","Copying text",copiedItemText];
-        subscribe(eventData);   
-    };
-    
-    const getSelectedText = () => window.getSelection()?.toString() ?? '';
-    
-    window.addEventListener("copy", processCopyEvent);
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
+/**
+ * Tracking text copied by the user.
+ *
+ * Adds event listener that monitors all copy events and sends text copied at the time as custom event.
+ */
+
+export default (subscribe, {filteredElements}) => {

```

Comment:
We've extended interface of that function which seems obligatory from now - we try to iterate using forEach on line 23, when nothing will be passed in func call it will fail. Also example in doc should be aligned for that: https://github.com/PiwikPRO/solutions-templates/blob/PPREVOPS-3158-remove-excessive-scroll/templates.js#L1655


Yea trying to use that ends up with: TypeError: Cannot read properties of undefined (reading 'filteredElements')

Reply:
fixed in the latest commit